src/modules/users/users.service.ts

```diff
@@ -449,7 +440,7 @@ export class UsersService {
       user.addressLine2 = addressLine2;
       user.state = state;
```

Comment:
Please add a country field to the address DTO. I will send 'Canada' as the value for this field.

Reply:
done